clients/python/test/e2e/_utils.py

```diff
@@ -31,7 +32,26 @@ def requires_dev_features(test):
         return pytest.mark.skip(
             (
                 f"{osparc.__version__=}<{str(repo_version)} "
-                "or {osparc_dev_features_enabled()=}"
+                f"or {osparc_dev_features_enabled()=}"
             )
         )(test)
     return test
+
+
+def requires_osparc_version(
+    *,
+    at_least: Optional[Version] = None,
+    at_most: Optional[Version] = None,
+    exactly: Optional[Version] = None,
+):
```

Comment:
I would add a check here to ensure that only one of them is not `None`.

Reply:
I think makes sense to allow multiple of them not to be `None`. E.g. one could have `at_least=Version("0.5.0"), at_most=Version("0.6.5")`